migrations/0014_create_events_v2.up.sql

```diff
@@ -0,0 +1,21 @@
+DROP TABLE IF EXISTS events;
+
+CREATE TABLE events (
```

Comment:
I think there may need to be a `capacity` here in addition to the `quantity` on ticket since the sum of available tickets may exceed the capacity of the event. e.g. the organizers may want to make available 75 regular tickets and 5 work trade entries but cap the event at 75 and not 80.